MimeKit/Cryptography/DefaultSecureMimeContext.cs

```diff
@@ -71,30 +71,22 @@ static DefaultSecureMimeContext ()
 		{
 			string path;
 
-#if !NETSTANDARD1_3 && !NETSTANDARD1_6
-			if (Path.DirectorySeparatorChar == '\\') {
+			if (Path.DirectorySeparatorChar is '\\') {
 				var appData = Environment.GetFolderPath (Environment.SpecialFolder.ApplicationData);
```

Comment:
I'm more inclined to leave the `==`. Need to think about this.

Reply:
the "is" operator nice for strings -- since it is an ordinal culture invariant comparison. Same as string.Equals("x", StringComparison.Ordinal);

For chars, codegen is the same -- but does unlock pattern matching against a variable name to enable:

c is 'a' or 'b'

If we utilize these patterns, we should consider utilizing is everywhere for consistency. I only changed a few right now to see how this settles with the project style. Let me know what you end up thinking.